src/microbe_stage/Microbe.Contact.cs

```diff
@@ -491,6 +460,16 @@ public bool CanEngulf(IEngulfable target)
         return EngulfSize > target.EngulfSize * Constants.ENGULF_SIZE_RATIO_REQ;
     }
 
+    /// <summary>
+    ///   Just like <see cref="ICellProperties.CanEngulf"/> but decoupled from Species and is based on the local
+    ///   condition of the microbe instead.
+    /// </summary>
+    /// <returns>True if this cell has all the requirements it needs for engulfing.</returns>
+    public bool CheckEngulfCapability()
```

Comment:
I think this should be a computed property, doesn't make sense to me to wrap this tiny bit of code in a method.

Reply:
Also why isn't this named `CanEngulf` like in the species, that seems a bit confusing to me to name the same thing multiple different things?

Reply:
Fixed.